io.edgehog.devicemanager.config.Telemetry.json

```diff
@@ -0,0 +1,14 @@
+{
+    "interface_name": "io.edgehog.devicemanager.config.Telemetry",
+    "version_major": 0,
+    "version_minor": 1,
+    "type": "properties",
+    "ownership": "server",
+    "mappings": [
+        {
+            "endpoint": "/request/%{interface_name}/periodS",
+            "type": "longinteger",
+            "description": "Set interval of period seconds between the end of the previous routine and the start of the next one. Set 0 to disable it."
```

Comment:
@rbino @harlem88 I'm not sure if 0 is a good choice, or if we rather should add one further endpoint for enabling it.